docs/src/main/paradox/client/details.md

```diff
@@ -59,5 +59,25 @@ Scala
 Java
 :  @@snip [GreeterClient.java](/plugin-tester-java/src/main/java/example/myapp/helloworld/LiftedGreeterClient.java) { #with-metadata }
 
+## Rich error model
 
+Beyond status codes you can also use the @link:[Rich error model](https://www.grpc.io/docs/guides/error/#richer-error-model) { open=new }. Currently there is no native support for this concept in akka-grpc. However you can use the following manual approach.
 
+Please follow @ref[this](../server/details.md) how to add the necessary dependency.
+
+Make the request and cast it as a `StatusRuntimeException`:
```

Comment:
I think we shouldn't recommend casting, as we can't be sure it'll always be a `StatusRuntimeException` - instead we could show how to check whether it's a StatusRuntimeException, and only in that case look for rich details.

Reply:
Jea that makes sense.